polaris-tokens/src/themes/dark.ts

```diff
@@ -14,6 +14,41 @@ export const metaThemeDarkPartial = createMetaThemePartial({
     'color-text-secondary': {value: colors.gray[11]},
     'color-bg-surface-secondary-active': {value: colors.gray[13]},
     'color-bg-surface-secondary-hover': {value: colors.gray[14]},
+    'color-bg-fill-transparent': {value: colors.whiteAlpha[8]},
+    'color-bg-fill-brand': {value: colors.gray[1]},
+    'color-text-brand-on-bg-fill': {value: colors.gray[15]},
+    'color-bg-surface-hover': {value: colors.gray[14]},
+    'color-bg-fill-hover': {value: colors.gray[14]},
+    'color-bg-fill-transparent-hover': {value: colors.whiteAlpha[9]},
+    'color-bg-fill-brand-hover': {value: colors.gray[5]},
+    'color-bg-surface-selected': {value: colors.gray[13]},
+    'color-bg-fill-selected': {value: colors.gray[13]},
+    'color-bg-fill-transparent-selected': {value: colors.whiteAlpha[11]},
+    'color-bg-fill-brand-selected': {value: colors.gray[9]},
+    'color-bg-surface-active': {value: colors.gray[13]},
+    'color-bg-fill-active': {value: colors.gray[13]},
+    'color-bg-fill-transparent-active': {value: colors.whiteAlpha[10]},
+    'color-bg-fill-brand-active': {value: colors.gray[4]},
+    'color-bg-surface-brand-selected': {value: colors.gray[14]},
+    'color-border-secondary': {value: colors.gray[13]},
+    'color-tooltip-border-tail-down-experimental': {
+      value: 'rgba(40, 40, 40, 1)',
+    },
+    'color-tooltip-border-tail-up-experimental': {
+      value: 'rgba(126, 126, 126, 1)',
+    },
```

Comment:
I have to hardcode these for now to match the shadow bevel on the dark fill

<img width="363" alt="Screenshot 2024-03-28 at 2 35 25 PM" src="https://github.com/Shopify/polaris/assets/20652326/00c1245c-b828-4325-b768-c8d57bd945c2">